scheduler/local.py

```
@@ -0,0 +1,397 @@
+            self.agent_name, self.path,
+        )
+
+    async def stop(self) -> None:
+        self._stopping = True
+        if self._task is None:
+            return
+        self._task.cancel()
+        try:
+            await self._task
+        except (asyncio.CancelledError, Exception):  # noqa: BLE001
+            pass
```

Ruff (0.15.12)

[warning] Use `contextlib.suppress(asyncio.CancelledError, Exception)` instead of `try`-`except`-`pass`

(SIM105)